kubelet/tests/test_kubelet.py

```diff
@@ -118,6 +118,12 @@
     "container_id://f69aa93ce78ee11e78e7c75dc71f535567961740a308422dafebdb4030b04903": ['pod_name:pi-kff76'],
     "kubernetes_pod_uid://12ceeaa9-33ca-11e6-ac8f-42010af00003": ['pod_name:dd-agent-ntepl'],
     "container_id://32fc50ecfe24df055f6d56037acb966337eef7282ad5c203a1be58f2dd2fe743": ['pod_name:dd-agent-ntepl'],
+    "container_id://a335589109ce5506aa69ba7481fc3e6c943abd23c5277016c92dac15d0f40479": [
```

Comment:
What do you thing of adding a dedicated test case to test the expected behaviour? (with a mocked tagger that returns `None` or `[]` and by asserting that metrics are not submitted) instead of appending logic to the existing functions.

Reply:
Yeah good point! Will do!

Reply:
🙇

Reply:
why do we add these 2 entries to `COMMON_TAGS`?